pkg/antctl/transform/common/transform.go

```diff
@@ -35,3 +36,42 @@ func GroupMemberPodTransform(pod networkingv1beta1.GroupMemberPod) GroupMemberPo
 	}
 	return GroupMemberPod{Pod: pod.Pod, IP: ipStr, Ports: pod.Ports}
 }
+
+type TableOutput interface {
+	GetTableHeader() []string
+	GetTableRow(args []string, maxColumnLength int) ([]string, error)
+}
+
+func GenerateTableElementWithSummary(list []string, maxColumnLength int) (string, error) {
```

Comment:
This is string processing function and is not likely have any error, is the returned error really needed? Maybe remove it as it cause all callers must handle it

Reply:
Deleted this error as well as the error in upstream functions.